catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/models/DisplayItemTest.scala

```diff
@@ -63,7 +64,8 @@ class DisplayItemTest extends FunSpec with Matchers {
   }
 
   it("correctly parses an Item without any locations") {
-    val item = JsonUtil.fromJson[Item]("""
+    val item =
+      ApiJsonUtil.fromJson[Item]("""
```

Comment:
It's not especially relevant to this patch, but why is there any JSON decoding happening in this test?

Reply:
very good question